listenbrainz/webserver/static/js/src/cb-review/CBReviewModal.tsx

```diff
@@ -84,7 +84,7 @@ export default class CBReviewModal extends React.Component<
           style={{ color: "black" }}
         />
       </span>
-      <ReactTooltip place="bottom" globalEventOff="click" clickable />
+      <ReactTooltip place="bottom" globalEventOff="click" clickable html={true} />
```

Comment:
To comply with lint:
```suggestion
      <ReactTooltip place="bottom" globalEventOff="click" clickable html />
```